qa/standalone/scrub/osd-scrub-snaps.sh

```diff
@@ -153,8 +153,6 @@ function create_scenario() {
     ceph-objectstore-tool --data-path $dir/${osd} "$JSON" clear-snapset clone_overlap || return 1
     JSON="$(ceph-objectstore-tool --data-path $dir/${osd} --head --op list obj11)"
     ceph-objectstore-tool --data-path $dir/${osd} "$JSON" clear-snapset clones || return 1
-    JSON="$(ceph-objectstore-tool --data-path $dir/${osd} --head --op list obj12)"
```

Comment:
@ronen-fr adding you as a reviewer specifically for this change. The command `head` was removed during one of the scrub refactors. Was there any reason, the `head` command was not removed from the test file?

Reply:
Leaving this question to Ronen.

Reply:
@Naveenaidu - what change are you referring to? The "head" option is still there, on line 3671 & 3740. ?

Reply:
Did you verify that the standalobne test still works? (I will be checking it now, anyway).

Reply:
@Naveenaidu - the line you have removed does not contain the 'clear_snapset' command!

Reply:
> Did you verify that the standalobne test still works? (I will be checking it now, anyway).

I did test the standalone tests and they were passing, apologies for not mentioning that in the description - thanks for checking it again though :)